onnxscript/function_libs/torch_aten/graph_building.py

```diff
@@ -247,38 +295,87 @@ def register_outputs(
         unwrapped_outputs = _unwrap_tensors_to_torch_values(outputs)
         if isinstance(unwrapped_outputs, torch.Value):
             self._graph.registerOutput(unwrapped_outputs)
-        else:
-            for ts_output in unwrapped_outputs:
-                assert isinstance(
-                    ts_output, torch.Value
-                ), f"ts_output must be a torch._C.Value, not {type(ts_output)}"
-                self._graph.registerOutput(unwrap_outputs)
+            return
+        assert isinstance(unwrapped_outputs, Sequence)
+        for ts_output in unwrapped_outputs:
+            assert isinstance(
+                ts_output, torch.Value
+            ), f"ts_output must be a torch.Value, not {type(ts_output)}"
+            self._graph.registerOutput(ts_output)
         return
 
+    def _add_constant_to_graph(self, constant) -> torch.Value:
+        if isinstance(constant, float):
+            return _create_torchscript_op(
+                self.graph,
```

Comment:
Should we have a standard on using self.graph or self._graph?

Reply:
We should. maybe just self._graph internal?

Reply:
Renamed graph to torch_graph